main.go

```diff
@@ -41,7 +44,24 @@ func ServePublicEndpoints(api *API, cfg *db.Config) {
 	log.Fatal(http.ListenAndServe(portString, mainHandler))
 }
 
+func loadDontScan() map[string]bool {
+	filepath := os.Getenv("DOMAIN_BLACKLIST")
```

Comment:
Could we make this optional by returning an empty map if DOMAIN_BLACKLIST isn't set?